.github/test.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+FACTOR=$1
+set -ex
+cd Polyhedron/demo
+LIST_OF_PLUGINS=$(for f in $(find ./Polyhedron/Plugins -iname "CMakeLists.txt" |egrep -v Three_examples); do\
+  egrep "polyhedron_demo_plugin" $f |cut -d\( -f2 | cut -d" " -f1|sort -u|egrep -v "partition_plugin"\
+ |egrep -v "basic_item_plugin"|egrep -v "vtk_plugin" |egrep -v "register_point_sets_plugin"| egrep -v "classification_plugin" | egrep -v "io_image_plugin" ||true; done)
```

Comment:
Why do you remove those plugins:
  - `partition_plugin`,
  - `basic_item_plugin`,
  - `vtk_plugin`,
  - `register_point_sets_plugin`,
  - `classification_plugin`, and
  - `io_image_plugin`?

Reply:
Some of them need specific dependencies that I don't want to install on a CI, others come from the Three examples that I don't really care about and need a specific CMake option that I didn't add, so the targets do not exist and the script crashes trying to build them.

Reply:
OK, I understand. That is to remove plugins that were not configured.

For that I would use the output of `make help`:
```shell
LIST_OF_PLUGINS=$(cmake --build . -t help | egrep 'plugin$' |& cut -d\  -f2)
```

That way you get only the targets known by the build system.

Reply:
I didn't know it was possible to do that. Thanks for the tip